scripts/run-httpd-with-fixtures

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 set -e
 
-REV=f1de61ad8897a845f2be49e3e2b2951bc678b678
+REV=aacacf872fceb6b051a940341a4ce71de10be0e8
```

Comment:
Can we bump this to `646376828d0520035246ccbbfcb82127160cd184`?

Reply:
Yeah I'm currently on it